client/client_api_test.c

```diff
@@ -0,0 +1,117 @@
+#include <stdlib.h>
```

Comment:
Can you move this file to its own `test` directory? See https://github.com/gwAdvNet2015/gw-kv-store/tree/master/server/test  and https://github.com/gwAdvNet2015/gw-kv-store/blob/master/server/Makefile#L43 for reference